lib/ocpp/v201/charge_point.cpp

```diff
@@ -714,6 +721,19 @@ void ChargePoint::init_websocket() {
     this->websocket->register_message_callback([this](const std::string& message) { this->message_callback(message); });
 }
 
+void ChargePoint::init_certificate_expiration_check_timers() {
+
+    // Timers started with initial delays; callback functions are supposed to re-schedule on their own!
```

Comment:
We only need a client certificate in case we use SecurityProfile 3, else I think we dont need to request it.
We only want to request a V2G certificate in case ISO15118V2GCertificateInstallationEnabled .